islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -69,7 +65,7 @@ function islandora_paged_content_pdf_batch_drush_command() {
       ),
       'namespace' => array(
         'description' => dt('The namespace for objects created by this command.  Defaults to namespace set in Fedora config.'),
-        'required' => FALSE,
+        'required' => TRUE,
```

Comment:
Previous behavior would be for it go through as a `NULL` on: https://github.com/discoverygarden/islandora_paged_content_pdf_batch/pull/20/files#diff-aedaa0eddfd8355f717175930d37881eR214 which causes the namespace to get inherited on: https://github.com/Islandora/islandora_batch/blob/7.x/includes/islandora_scan_batch.inc#L194-L199 which then does some processing on: https://github.com/Islandora/islandora_batch/blob/7.x/includes/preprocessor_base.inc#L139-Lundefined.

If the Drush command outright wasn't working what version of Drush are you running? May be another issue at play.